modules/certificate/certificates.permission.js

```diff
@@ -9,6 +9,7 @@ const certificatePermissionsQuery = {
 };
 
 const certificatePermissionsMutations = {
+  generateCertificate: hasRoles([USER, ADMIN, SUPERADMIN]),
```

Comment:
you need to add input validation rule, also hasRoles rule is useless here

Reply:
you need to use other middlewares that hasRoles uses inside itself, but hasRoles check is useless here because it always returns true

Reply:
Ok, I will fix it